Jenkinsfile_CNP

```diff
@@ -0,0 +1,10 @@
+#!groovy
+
+@Library("Infrastructure")
+
+def type = "java"
+def product = "draft-store"
+def app = "draft-store"
```

Comment:
Internally in the pipeline, `product` and `app` get concatenated together to name the infrastructure so might be worth changing app to `service` or `backend` or something like that so the app name in the ase will be e.g. `draft-store-service-prod` rather than `draft-store-draft-store-prod`